### ISSUE TITLE
Telegram Bot not working correctly on "mark available"

### ISSUE THREAD

**Issue by Bischi1997** [tidusjar/Ombi#3900]
**Describe the bug**
If a new Movie or Serie is added and downloaded. Once it is available, no message gets sent to Telegram.
Testing as well as the "New Request" Notification work.
Notification to the mobile works.

**To Reproduce**
1. Go to a 'Requests'
2. Go to a 'Available Requests'
3. Go to a 'Details' of Series
4. Click on 'Requests'
5. Click on 'Mark Unavailable'
6. Click on 'Mark Available'
7. No Message gets sent
8. See error in the Logfile

**Expected behavior**
The message for "Request Available" gets sent.

**Logs (Logs directory where Ombi is located)**
info: Ombi.Notifications.Agents.LegacyMobileNotification[0]
      there are no users to send a notification for RequestAvailable, for agent Mobile
info: Ombi.Notifications.Agents.MobileNotification[0]
      there are no users to send a notification for RequestAvailable, for agent Mobile
fail: Ombi.Api.Api[1000]
      StatusCode: BadRequest, Reason: Bad Request, RequestUri: https://api.telegram.org/botAPI/sendMessage

**Desktop (please complete the following information):**
 - OS: Linux

**Ombi Version (please complete the following information):**
- Version 4.0.731.0
- Media Server: Plex
- Database Type: SQLite

**Comment by costaht** [comment]
I think I can guess what the problem is because I was about to open an Issue for that too.
Do you have the tag **{PosterImage}** in your notifications? If yes, remove it and test with TV shows again. This behavior only happens for TV Shows notifications and it's very annoying.
Let me know.

**Comment by Bischi1997** [comment]
@costaht Just tested it without the "{PosterImage}". It works now, so it is seems to be a bug.

**Comment by costaht** [comment]
Yep, and here's the log. Line 14 is the error we get when Ombi tries to Notify _New Request_ for TV Shows while using `{PosterImage}`.
https://paste.centos.org/view/84b23594
Probably is just a semi-colon missing :)

**Comment by Xarano-GIT** [comment]
+1 on this issue, I would love to be able to add poster images back into my notifications to add that little extra!

**Comment by bernarden** [comment]
Anyone tested this recently? It's working for me.
